app/extensions/__init__.py

```diff
@@ -193,7 +194,7 @@ class JSON(db.TypeDecorator):
     def process_bind_param(self, value, dialect):
         # Adapted from sqlalchemy/sql/sqltypes.py JSON.bind_processor
         def json_serializer(*args, **kwargs):
-            return json.dumps(*args, **kwargs, cls=flask.json.JSONEncoder)
+            return json.dumps(*args, **kwargs, cls=CustomJSONEncoder)
```

Comment:
Curious as to why you made this change.

Reply:
I added the decimal fields for AssetGroupSighting encounter latitude and longitude... but I think we might be using float elsewhere... I don't know if it matters :thinking:

Reply:
(decimal fields are not dealt with in `flask.json.JSONEncoder`)

Reply:
Wow, that is a surprise to have anything cope with floats but not integers, bu yeah I understand the change.

Reply:
No it's not integers, it's https://docs.python.org/3/library/decimal.html, so it's like float but doesn't lose precision

Reply:
Oh I just talked to @naknomum and he said java (edm) uses double, which has the same precision as python float, so we'll just use float.